web/packages/teleterm/package.json

```diff
@@ -10,9 +10,10 @@
   },
   "license": "Apache-2.0",
   "scripts": {
-    "start": "electron-vite dev",
-    "build": "electron-vite build",
+    "start": "pnpm build-wasm && electron-vite dev",
+    "build": "pnpm build-wasm && electron-vite build",
```

Comment:
This is not ideal, we will build WASM twice in the tag build. It would be good to make it smarter so that it skips the build if the Rust source code didn't change.

Reply:
Yeah, it also adds extra overhead to just starting the app in dev mode. I think it's fine to commit it as is for now.